packages/lesswrong/components/posts/PostsPreviewTooltip.tsx

```diff
@@ -113,7 +113,7 @@ const styles = (theme: ThemeType): JssStyles => ({
 const getPostCategory = (post: PostsBase) => {
   const categories: Array<string> = [];
 
-  if (post.isEvent) categories.push(`Event`)
+  if (post.isEvent) return null
```

Comment:
@b0b3rt @darkruby501 Changes in two files in this PR will affect LW. I'll comment on them in github.

In this file, I'm cleaning up the hoverover post preview for events. They already have [Event] in the title, so I didn't think they needed to say "Event" under the title. I also removed the word count for events.

Instead, I put the event time and location, which seemed like the most relevant info. Ex:
<img width="468" alt="Screen Shot 2023-08-07 at 6 22 25 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/2496eaac-188c-40ce-9f6e-402b66d33ed3">